src/jaxsim/api/model.py

```diff
@@ -50,6 +50,10 @@ class JaxSimModel(JaxsimDataclass):
     def description(self) -> jaxsim.parsers.descriptions.ModelDescription:
         return self._description.get()
 
+    contact_model: jaxsim.rbda.ContactModel | None = dataclasses.field(
+        default=None, repr=False, compare=False, hash=False
+    )
+
```

Comment:
Any reason to add this attribute after the property? Can you please move it after `terrain`?

Reply:
Also, compare and hash are disabled class-wise, it's no longer necessary to disable them in the field definition.

Reply:
I might have messed something while rebasing. Fixed in https://github.com/ami-iit/jaxsim/pull/178/commits/882a4a0ae04e8028c3ff6b0ab27dbaea43779826